cmake/Modules/Findlibunwind.cmake

```diff
@@ -0,0 +1,29 @@
+# - Try to find libunwind
+# Once done, this will define
+#
+#  libunwind_FOUND - system has libunwind
+#  libunwind_INCLUDE_DIRS - the libunwind include directories
+#  libunwind_LIBRARIES - link these to use libunwind
+
+include(LibFindMacros)
```

Comment:
It looks like the source for this macro is here: https://github.com/Tronic/cmake-modules/blob/master/LibFindMacros.cmake . Is the license compatible? If so, does it make sense to update the file in GR to be closer to the current version?

Reply:
Why don't we use this module more? It's really cool!

Reply:
Can you please check if the license is compatible? I'm not even sure what that means in the CMake context. Do we need to copy this into the source tree?

Reply:
The files says it is public domain, so should be fine.